homeassistant/exceptions.py

```diff
@@ -45,11 +66,7 @@ def __init__(
             translation_key=translation_key,
             translation_placeholders=translation_placeholders,
         )
-        self._message = message
-
-    def __str__(self) -> str:
-        """Return exception message string."""
-        return self._message
+        self._translate = bool(translation_domain and translation_key)
```

Comment:
I don't understand this change. The constructor still requires a message, so why would we discard that and instead return a translated message?

We could maybe make `ConfigValidationError.__init__` kwarg only, in a separate PR.

Reply:
Reverted the changes to `ConfigValidationError`